### ISSUE TITLE
Savage bite weird GCD interaction bug

### ISSUE THREAD

**Issue by GrizB37** [slowtorta/turtlewow-bug-tracker#4295]
**Please, provide a clear description what the bug is:**
Savage Bite should be off the gcd, but it’s currently causing gcd to everything; even including Maul for some reason. Making it a threat loss to be used at higher tier of raiding as it often costs you a Maul to use SB
**Steps to reproduce the behavior with as much detail as possible:**

1. Use savage bite
2. Notice all your spells including ones that never should get triggered onto GCD and now on GCD. 
3. Lose threat. 

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

NPC: https://database.turtle-wow.org/?npc=
Item: https://database.turtle-wow.org/?item=
Quest: https://database.turtle-wow.org/?quest=
Spell: https://database.turtle-wow.org/?spell=
Object: https://database.turtle-wow.org/?object=

**Expected behavior. Describe how it should work:**
Basically savage bite should not be triggering GCD on things like maul which should never be effected by GCD.

**Comment by cyaohiri** [comment]
Related to #3442, #4142, and #1664.
